File: CursoPOOUber/Java/Driver.java
package Java;

class Driver extends Account{
    public Driver(Integer id, String name, String document, String email, String password){
        super(name, document, email, password);
    }

    void PrintDataDriver(){
        System.out.println("Name Driver: " + name + "\nDocument Driver: " + document + "\nEmail Driver: "
        + email + "\nPassword Driver: " + password);
    }
    
}

File: CursoPOOUber/JS/Driver.js
class Driver extends Account{
    constructor(name, document, email, password){
        super(name, document, email, password)

    }
    printDataDriver(){
        console.log(this.name);
        console.log(this.document);
        console.log(this.email);
        console.log(this.password);
    }
    
}
